gui/src/app/prototypes/AnalysisPyPrototype/analysisPyWorker.ts

```diff
@@ -0,0 +1,91 @@
+import { PyodideInterface, loadPyodide } from "pyodide";
+
+let pyodide: PyodideInterface | null = null;
+const loadPyodideInstance = async () => {
+  if (pyodide === null) {
+    const p = await loadPyodide({
+      indexURL: "https://cdn.jsdelivr.net/pyodide/v0.26.1/full",
+      stdout: (x: string) => {
+        self.postMessage({ type: "stdout", data: x });
+      },
+      stderr: (x: string) => {
+        self.postMessage({ type: "stderr", data: x });
+      },
+    });
+    pyodide = p;
+    await pyodide.loadPackage(["numpy", "matplotlib"]);
```

Comment:
Passing these as the `packages` argument to the loadPyodide should be faster:

> A list of packages to load as Pyodide is initializing. This is the same as loading the packages with [pyodide.loadPackage()](https://pyodide.org/en/stable/usage/api/js-api.html#pyodide.loadPackage) after Pyodide is loaded except using the packages option is more efficient because the packages are downloaded while Pyodide bootstraps itself.

Reply:
Done.